input/fsh/omrs-observation.fsh

```diff
@@ -15,19 +15,34 @@ Description: "Observation resource created by a OpenMRS that representing a resu
 * encounter 0..1
 * value[x]  0..1
 
-// OpenMRS Task  Example
+/* OpenMRS Observation Resource  Example */
+
 Instance: example-openmrs-Observation
 InstanceOf: OpenMRSObservation
 Usage: #example
 Title: "Openmrs Observation"
 Description: "Example OMRS Observation Resource"
 * identifier.value = "e8f8d71a-6c14-4105-b6fe-30f3cf6b7dc9"
```

Comment:
I don't think we map an identifier field in the translator: https://github.com/openmrs/openmrs-module-fhir2/blob/master/api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/ObservationTranslatorImpl.java